package.json

```diff
@@ -177,6 +190,7 @@
         ]
       },
       "additionalLockfilePaths": [
+        "azure",
         "common/build/build-common",
```

Comment:
(Not for this PR) do the packages under `common/lib` need to be added here? For my knowledge, what is this field used for?

Reply:
See #12726.

Reply:
The azure entry shouldn't be needed. I'll investigate.

Reply:
Seems related to switching the azure release group from a "simple" config to a "complex" one (an object instead of a string). I'm going to ignore for this PR since there's a clear workaround (adding the entry here) and follow up with another change later.